build_tfm.py

```diff
@@ -349,6 +349,13 @@ def _copy_binaries(source, destination, toolchain, target):
 
     if "TFM_V8M" in TARGET_MAP[target].extra_labels:
         install_dir = abspath(join(source, os.pardir, os.pardir))
+
+        # Support multi-level TF-M target name say "nuvoton/m2354"
```

Comment:
This would imply for any multi-level target name, so please change the comment
``` suggestion
# Support multi-level TF-M target name
```

Reply:
Updated comment above and commit message.